environment.py

```diff
@@ -168,3 +179,45 @@ def after_step(context, step):
             file_logger.info('Ansible Output: %s' % context.result)
             print('Ansible Output: %s' % context.result)
 
+
+def before_feature(context, feature):
```

Comment:
I'd like to see a little bit of comments here explaining what this function is trying to do.  I managed to figure it out, but some description would be helpful.